pkg/agent/openflow/pipeline.go

```diff
@@ -695,7 +695,7 @@ func (f *featurePodConnectivity) conntrackFlows() []binding.Flow {
 				MatchCTStateNew(true).
 				MatchCTStateTrk(true).
 				Action().CT(true, ConntrackCommitTable.GetNext(), f.ctZones[ipProtocol], f.ctZoneSrcField).
-				MoveToCtMarkField(PktSourceField, ConnSourceCTMarkField).
+				MoveToCtMarkFields(regFieldToCtMarkFieldList...).
```

Comment:
We have single case anyway, I feel introducing a new struct RegFieldToCtMarkField is not more readable than original code. Could we leave it to future when it's really necessary?

Reply:
Removed it.